HW_1/task_1.py

```diff
@@ -0,0 +1,13 @@
+#Найдите сумму цифр трехзначного числа.
+#*Пример:*
+#123 -> 6 (1 + 2 + 3)
+#100 -> 1 (1 + 0 + 0) 
+
+a = input("Введите трехзначное число ")
+a = int(a)
```

Comment:
Это можно совместить.
```suggestion
a = int(input("Введите трехзначное число "))
```